inc/3rd-party/nextgen-gallery/nextgen-gallery.php

```diff
@@ -19,6 +19,7 @@
 
 Imagify_NGG::get_instance()->init();
 Imagify_NGG_DB::get_instance()->init();
+Imagify_NGG_Dynamic_Thumbnails_Background_Process::get_instance();
```

Comment:
what's the use of this?

Reply:
Both `WP_Background_Process` and `WP_Async_Request` launch hooks when instantiated.

Reply:
the instanciation in `imagify_ngg_maybe_add_dynamic_thumbnail_to_background_process` is not enough?

Reply:
No, they launch ajax callbacks and cron schedules, this must be done all the time.